### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 016c122af9d85da32726d9bb0d2b318eda06c2e8 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/71687/016c122af9d85da32726d9bb0d2b318eda06c2e8/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/71687/016c122af9d85da32726d9bb0d2b318eda06c2e8/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ tests/ci/cancel_and_rerun_workflow_lambda/app.py
 - Backward Incompatible Change
 - Build/Testing/Packaging Improvement
 - Documentation (changelog entry is not required)
-- Critical Bug Fix (crash, LOGICAL_ERROR, data loss, RBAC)
```
Comment (nikitamikhaylov) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Why?

Comment (nikitamikhaylov) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Ah, I see: https://github.com/ClickHouse/ClickHouse/pull/71679

Comment (Algunenano) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: I still don't see why. This was introduced to make sure some PRs were always backported (the ones related to logical errors and crashes). Now we'll go back to missing backports

Comment (Algunenano) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: But we still need to backport the fixes. We can change the category to call it `Critical bugfixes and logical errors` if that's better

**Pull request merged**